testing/esp/wudetails/key/dict1_hthor_testcase1.json

```diff
@@ -459,23 +459,23 @@ testCase(ScopeFilter={
                             'CreatorType': None
                         },
                         {
-                            'Name': 'Definition',
+                            'Name': 'DefinitionList',
                             'RawValue': '{masked}',
                             'Formatted': '{masked}',
                             'Measure': None,
                             'Creator': '{masked}',
                             'CreatorType': None
                         },
                         {
-                            'Name': 'EclName',
-                            'RawValue': 'dict1',
-                            'Formatted': 'dict1',
+                            'Name': 'EclNameList',
+                            'RawValue': '["dict1"]',
+                            'Formatted': '["dict1"]',
                             'Measure': None,
                             'Creator': '{masked}',
                             'CreatorType': None
                         },
                         {
-                            'Name': 'Definition',
+                            'Name': 'DefinitionList',
```

Comment:
Note - the repeated attributes may not occur next to each other.  In this case this definition should have been combined with the attribute on line 462.